### ISSUE TITLE
Ethereal Manor Spawn always appear in adventur even if it's not present.

### ISSUE THREAD

**Issue by ExoZNiK** [Razzmatazzz/RemnantSaveGuardian#207]
Just wanted to notify that in the adventure analyzer it keeps showing "Ethereal Manor" as injectible of the "Forgotten Null" location even if it's not present. It keep appear in every reroll of the adventure no matter what locations are spawning.
Just to cit. the wiki about that, it could be implemented to be shown only if the location with burning streets are present in the adventure, which are: Cotton's Kiln
, Butcher's Quarter
, Sunken Haunt
, Derelict Lighthouse.

Also don't know what "enter the dream" missing quest item is talking about, maybe that you can get the anguish?

**Comment by identity7** [comment]
Yes. "Enter the dream" is a questline for the dranception event. it disappears from RSG once u complete it

**Comment by Skynetexe** [comment]
Yeah, I have the same issue here

**Comment by Auricrystal** [comment]
yeah this is a parse mistake, Forgotten Null and Consecrated Throne are both "Global Events" like Huntress dream and root walker dream, since you can use their consumable orbs anywhere they ALWAYS are loaded in the game save so Save Analyzer just displays them even if the actual event isnt there, since you can be holding onto the consumable after rerolling.

Game/Content/World_DLC1/Quests/Quest_Injectable_BurningCity_DLC.Quest_Injectable_BurningCity_DLC_C should be the "real" event that needs to be parsed, and the SpawnTable should be ignored since it always is next to the Forgotten Null thus acts as a false positive.

**Comment by ExoZNiK** [comment]
So after a person get the Anguish weapon is normal that "Enter the Dream" keep being displayed in missing items under quests? Just like Mnemonic core II and Decor code?
